### PR TITLE
Update java-tracer docs for 0.10.0 release

### PR DESCRIPTION
**Pull request by realark**
### What does this PR do?
- Docs for MDC usage
- Beta instrumentation list update

### Preview link
https://github.com/DataDog/documentation/blob/ark/java-0.10.0-update/content/tracing/setup/java.md

### CODE REVIEW

File: content/tracing/setup/java.md [hunk 1]
```diff
@@ -255,6 +255,56 @@ public class MyClass {
 }
 ```
 
+## Logging and MDC
+
+The Java tracer exposes two api calls to allow printing trace and span identifiers along with log statements, `CorrelationIdentifier#getTraceId()` and `CorrelationIdentifier#getSpanId()`.
```
Comment (cswatt) [inline content/tracing/setup/java.md]: capitalize API, use the serial comma: 
The Java tracer exposes two API calls to allow printing trace and span identifiers along with log statements, `CorrelationIdentifier#getTraceId()`, and `CorrelationIdentifier#getSpanId()`.

Comment (realark) [reply content/tracing/setup/java.md]: 👍 Changes pushed.

**Comment by phrawzty** [comment]
Should this be merged now, or are we waiting for the software to be released, or…?

**Comment by realark** [comment]
@phrawzty Sorry, yes this is ready to go. I'll note that next time I submit a PR :-). Thanks!

**Pull request merged**